packages/core/src/App/Containers/PasswordSelectionModal/password-selection-modal.jsx

```diff
@@ -28,22 +28,22 @@ const PasswordSelectionModal = observer(
         const { ui } = useStore();
         const { is_mobile } = ui;
 
+        const loggedIn = !!cacheTrackEvents.parseCookies('client_information');
         React.useEffect(() => {
-            trackEventWithCache({
-                name: 'ce_virtual_signup_form',
-                properties: {
-                    action: 'password_screen_opened',
-                    form_name: is_mobile ? 'virtual_signup_web_mobile_default' : 'virtual_signup_web_desktop_default',
+            cacheTrackEvents.track(
+                {
```

Comment:
cacheTrackEvents.loadEvent can be used.Triggers an event when the page is loaded and immediately stores it in the cache if analytics is not yet ready.

Reply:
Fixed.